drivers/pktmem.c

```diff
@@ -16,17 +17,17 @@
 
 #include <stdio.h>
 #include <string.h>
+#include <arch.h>
 
 
 #define CACHE_LINE_SIZE 64
-#define PAGE_SIZE 4096
 
 #define TRUE_LRU 0
 
 
-#define PKT_BUF_SIZE (2048 - CACHE_LINE_SIZE)
-#define PKT_BUF_CNT	(size_t)((PAGE_SIZE - CACHE_LINE_SIZE) / PKT_BUF_SIZE)
-#define PKT_BUF_IDX	11	// log2(PAGE_SIZE) - ceil(log2(PKT_BUF_CNT))
+#define PKT_BUF_SIZE       (2048 - CACHE_LINE_SIZE)
+#define PKT_BUF_CNT        (size_t)((_PAGE_SIZE - CACHE_LINE_SIZE) / PKT_BUF_SIZE)
+#define PKT_BUF_IDX        11 /* log2(_PAGE_SIZE) - ceil(log2(PKT_BUF_CNT)) */
```

Comment:
We don't know it's value and use definit oo on from arch.h, because it's target dependent. We need a value provided by arch for mmap/munap to work properly

Reply:
yes, but the the reason for it being exactly 11 is that we assumed page_size == 4096 at least acoording to the comment